src/components/notebook-utils.js

```diff
@@ -437,17 +572,30 @@ export const cohortNotebook = cohortName => `
    "metadata": {},
    "outputs": [],
    "source": [
-    "plt.rcParams.update({'font.size': 14})\\n",
+    "col_to_plot = cohort.columns[0]\\n",
+    "for col in cohort.columns:\\n",
+    "    if col.endswith(('id', 'Id', 'ID')):\\n",
+    "        continue\\n",
+    "    if cohort[col].count() != 0:  # Plot a non-empty column.\\n",
+    "        col_to_plot = col\\n",
+    "        break\\n",
     "\\n",
-    "# Drop columns with 'ID' or 'id'\\n",
-    "cohort_to_plot = cohort[cohort.columns.drop(list(cohort.filter(regex='id|ID')))]\\n",
-    "cohort_first_column = cohort_to_plot[cohort_to_plot.columns[0]]\\n",
+    "title = f'Plot of {col_to_plot} for cohort \"{COHORT}\"'\\n",
```

Comment:
Assuming you want an `\"` as the output in the file, you need to escape the backslash as well: `\\"`. This is also on line 965. That should make the lint pass.

Reply:
thx!